src/components/DepartmentsList.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import { getDepartments } from '../api/api';
+import Department from './Department';
+import Loader from './Loader';
+
+export default class DepartmentsList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      departments: [],
+      isLoading: false,
+      error: null,
+    };
+  }
+  async componentDidMount() {
+    this.setState({ isLoading: true });
+    setTimeout(async () => {
```

Comment:
Why do you use setTimeout here? Do you have some specific task for this?

Reply:
I used the setTimeout to show how the spinner would work

Reply:
Got it, but i think it may be removed)))